CMake/Modules/UrhoCommon.cmake

```diff
@@ -605,10 +605,15 @@ else ()
             set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${ARM_CFLAGS}")
             set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${ARM_CFLAGS}")
         else ()
-            if (NOT XCODE AND NOT WEB)
+            if (NOT XCODE AND NOT WEB AND NOT E2K)
                 set (CMAKE_C_FLAGS "-mtune=generic ${CMAKE_C_FLAGS}")
                 set (CMAKE_CXX_FLAGS "-mtune=generic ${CMAKE_CXX_FLAGS}")
             endif ()
+            if (E2K)
+                # MCST lcc compiler use =native option instead =generic
+                set (CMAKE_C_FLAGS "-mtune=native ${CMAKE_C_FLAGS}")
+                set (CMAKE_CXX_FLAGS "-mtune=native ${CMAKE_CXX_FLAGS}")
+            endif()
```

Comment:
Try remove these lines and see if everything still works. Thanks.

Reply:
In this case (after remove these lines), optimization flags in CMake configuration will be as follows:
-- CFLAGS: -msse3 -pthread -fdiagnostics-color=auto
(without "-mtune=native")

With this options compilation is done without any problems, everything still works.

Reply:
Perfect. That’s what I expected. Can you push that commit. Thanks!

Reply:
Need to remove these lines? In this case, for e2k architecture, "-mtune=native" option will be lost.

Reply:
Isn’t all compiler emit native instruction sets by default?

Reply:
Assuming “cc” is an alias to your compiler, if you would do a simple build like so, “cc HelloWorld.c”, would it generate a native binary?

Reply:
Yes, in that case it generate a binary file.

By default, MCST lcc compiler uses  "-march=elbrus-vX" option, where X is version of processor command system (by default, this option takes value corresponding to model of the processor on which the compiler is currently running).
For example, for the Elbrus-4C (4 core, 750MHz) processor option will be "-march=elbrus-v3", and for the Elbrus-8C (8 core, 1300 MHz) processor option will be "-march=elbrus-v4".

Reply:
I wanted, by default, to include additional optimizations ("-mtune=native" option) for the processor on which compilation will go. But probably it’s better not to do that 😄

Reply:
I decided to recreate PR (in which I remove these lines).
Sorry for my perfectionism 😄 Many thanks!